python/lsst/ip/isr/ptcDataset.py

```diff
@@ -626,6 +641,10 @@ def fromTable(cls, tableList):
             else:
                 inDict['noiseMatrix'][ampName] = record['NOISE_MATRIX']
                 inDict['noiseMatrixNoB'][ampName] = record['NOISE_MATRIX_NO_B']
+            if calibVersion < 1.5:
+                inDict['covMatrixSideFullCovFit'] = inDict['covMatrixSide']
+            else:
+                inDict['covMatrixSideFullCovFit'] = record['COV_MATRIX_SIDE_FULL_COV_FIT']
```

Comment:
Okay, I see why I originally thought this wouldn't work.  This is matched to that `COV_MATRIX_SIDE` above, and actually will get overwritten through the loop of amps (but should be the same).  Maybe worth adding a comment here about this.